installers/centos.sh

```diff
@@ -583,7 +699,11 @@ fi
 ls /etc/php.d | grep "snowflake"
 if (($? >= 1)); then
   yum update
-  yum install -y gcc cmake php-pdo php-json php-dev
+  if ((CURRENT_OS == 7)); then
+    yum install -y gcc cmake php-pdo php-json php-dev
+  else
+    yum install -y gcc cmake php-pdo php-json
```

Comment:
why no php-dev for CentOS 8?

Reply:
the package php-dev doesnt exist. in centos7 this if it cant find a package, the script just moves on, but in centos8 it will just stop the script at that point.
I was a little concerned about just removing it, but I guess we can?

Reply:
@tomonorman I think that is just the wrong package name. it is called php-devel on centOS. Can you research it?

Reply:
yup you're right, this is a centos/fedora, debian/ubuntu thing, will puish up the change